data/resources/plugins/adddir.py

```
@@ -64,19 +64,20 @@ def filechooser_response(dialog, response, clsdlg):
                         btarget = srvutl.filename_normalize(source)
                         target = os.path.join(srvrepo.docs, btarget)
                         srvutl.filename_import(source, target)
-            dialog.destroy()
 
         window = self.app.get_widget('window')
         clsdlg = MiAZFileChooserDialog(self.app)
         filechooser_dialog = clsdlg.create(
-                    parent=window,
+                    enable_response=True,
                     title=_('Import a directory'),
```

Ruff (0.8.2)

Undefined name `_`

(F821)